rele/subscription.py

```diff
@@ -33,25 +41,32 @@ def set_prefix(self, prefix):
 
     @property
     def filter_by(self):
-        return self._filter_by
+        return self._filters
 
     def set_filter_by(self, filter_by):
```

Comment:
Seems to me this method should now be called `set_filters`

Reply:
Same here, but kept it because parameter is still called `filter_by`.

Changing it.